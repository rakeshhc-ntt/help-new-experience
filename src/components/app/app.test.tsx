import React from 'react'
import { render } from '@testing-library/react'
import App from './app'

it('matches snapshot', () => {
  const { asFragment } = render(
    <App>
      <h1>Hello World</h1>
    </App>
  )
  expect(asFragment()).toMatchSnapshot()
})
