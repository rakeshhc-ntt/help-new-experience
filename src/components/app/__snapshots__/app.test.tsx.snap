// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="main-wrapper"
  >
    <h1>
      Hello World
    </h1>
    <style>
      
        .main-wrapper {
          max-width: 620px;
          margin: auto;
          margin-top: 30px;
          padding: 15px;
        }
      
    </style>
  </div>
</DocumentFragment>
`;
